### PR TITLE
Setup Docker build & deploy

### PR DESCRIPTION
**Pull request by akrava**
Closes: #13

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,13 +15,13 @@ jobs:
                 node-version: 13.6.0
 
           - name: Lint
-            run: echo "FIXME(add linting)"
+            run: cd frontend && npm ci && npm run-script lint
```
Comment (Veetaha) [inline .github/workflows/ci.yml]: I think it is more common to use `npm run` instead of `npm run-script` today...

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frontend
+FROM node:12.14.1 AS frontend
```
Comment (Veetaha) [inline Dockerfile]: Please, keep CI and docker node versions in sync.

File: bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+PORT_DOCKER=3000
+PORT_LOCAL=8080
+PROJECT_NAME="int20htest"
```
Comment (Veetaha) [inline bootstrap.sh]: I thought our project name was `audio-guesser`

File: bootstrap.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+docker build -t $PROJECT_NAME .
+docker run --env PORT=$PORT_DOCKER -p $PORT_LOCAL:$PORT_DOCKER -d $PROJECT_NAME
+echo "App run on: 127.0.0.1:$PORT_LOCAL"
```
Comment (Veetaha) [inline bootstrap.sh]: `App is running on ...`

File: frontend/karma.conf.js [hunk 5]
```diff
@@ -1,6 +1,8 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```
Comment (Veetaha) [inline frontend/karma.conf.js]: What was the purpose of the changes in this file?

Comment (akrava) [reply frontend/karma.conf.js]: The purpose was to make tests runnable without errors

**Pull request merged**